lighthouse-core/test/audits/seo/font-size-test.js

```diff
@@ -18,14 +19,21 @@ const validViewport = 'width=device-width';
 
 describe('SEO: Font size audit', () => {
   const makeMetaElements = viewport => [{name: 'viewport', content: viewport}];
-  const getFakeContext = () => ({computedCache: new Map()});
+
+  /** @param {LH.SharedFlagsSettings['formFactor']} formFactor */
+  const getFakeContext = (formFactor = 'mobile') => ({
```

Comment:
kinda yah but it gets a little weird. in addition the arguments, it also consumes the audit's `(default)options` object.  

centralizing as a utility method does DRY things up quite a bit, but it adds complexity and reduces overall readability. so i'm prone to keep it as is. I asked @patrickhulce and he was thinking the same.

Reply:
> centralizing as a utility method does DRY things up quite a bit, but it adds complexity and reduces overall readability

This isn't important for this PR, but I'm actually with @adamraine's original comment and disagree with this in for audit contexts in particular (in the general case of generating test data I might lean more towards this statement).

If the goal is to eventually type check all our tests (see #11728), it would generally be better to have audit tests have an easy way to get a fakeContext that's fully filled out, rather than each one having their own subset of a `Partial<>` and hope that it's the right subset and not just accidentally hitting the right side of falsy checks.

Audit context is mostly made up of simple readonly objects anyways, except for `computedCache`, which nothing should be manually touching anyways. It would be easy to have a `getFakeAuditContext(settings: RecursivePartial<LH.Config.Settings>, options?: Record<string, any>}` or whatever that gives back a real auditContext, and would cover all the fakeContext cases being touched in this PR, at least.